packages/webapp/components/FooterNavBar.tsx

```diff
@@ -27,7 +27,7 @@ const selectedMapToTitle: Record<keyof UseActiveNav, string> = {
   home: 'Home',
   search: 'Search',
   bookmarks: 'Bookmarks',
-  notifications: 'Notifications',
+  notifications: 'Activity',
```

Comment:
Where did this decision come from, missing the context

Reply:
![image](https://github.com/dailydotdev/apps/assets/1681525/73ca18b2-64a3-4e1f-a083-166874a4d169)
From the design

Reply:
Ah ok :) missed it before then

Reply:
Me too 🙈